portal-api/api-tests/v1/deals/deal-clone.api-test.js

```diff
@@ -92,14 +92,15 @@ describe('/v1/deals/:id/clone', () => {
       let createdFacilities;
 
       beforeEach(async () => {
-        const { body } = await as(anHSBCMaker).post(dealToClone).to('/v1/deals');
-        originalDeal = body;
-        const originalDealId = originalDeal._id;
-
+        const { body: createdDeal } = await as(anHSBCMaker).post(dealToClone).to('/v1/deals');
+        const originalDealId = createdDeal._id;
         createdFacilities = await createFacilities(anHSBCMaker, originalDealId, completedDeal.mockFacilities);
+
+        const { body: dealAfterCreation } = await as(anHSBCMaker).get(`/v1/deals/${originalDealId}`);
+        originalDeal = dealAfterCreation.deal;
       });
 
-      it('clones a deal with only specific properties in `details`, wipes `comments`, `editedBy` `ukefComments`, `specialConditions`, changes `maker` to the user making the request, marks status `Draft`', async () => {
+      it.only('clones a deal with only specific properties in `details`, wipes `comments`, `editedBy` `ukefComments`, `specialConditions`, changes `maker` to the user making the request, marks status `Draft`', async () => {
```

Comment:
.only

Reply:
Good spot! Was from debugging. I've removed this and updated/corrected the other tests here: https://github.com/UK-Export-Finance/dtfs2/pull/326/commits/d309ae92e51b360b5ab14bc6aca061b6587be38d